### PR TITLE
Build and release for linux arm64

### PR DESCRIPTION
**Pull request by vcheung-stripe**
### Reviewers
cc @stripe/developer-products

 ### Summary

Update goreleaser config to build and release for linux arm64. Will need to verify if it works with package managers.

### CODE REVIEW

File: .goreleaser/linux.yml [hunk 1]
```diff
@@ -17,6 +17,17 @@ builds:
       - linux
     goarch:
       - amd64
+  - id: stripe-linux-arm
+    ldflags:
+      - -s -w -X github.com/stripe/stripe-cli/pkg/version.Version={{.Version}}
+    binary: stripe
+    env:
+      - CGO_ENABLED=0
```
Comment (etsai-stripe) [inline .goreleaser/linux.yml:25]: is there a reason why `CGO_ENABLED` is off?

Comment (vcheung-stripe) [reply .goreleaser/linux.yml:25]: `goreleaser build -f .goreleaser/linux.yml` fails on my local machine unless `CGO_ENABLED` is off. I'm not really sure what the implications are.

**Comment by etsai-stripe** [comment]
wondering if we should release it separately from your other PR in case this doesnt work for jfrog or dockers. but i could be just over-thinking

**Comment by vcheung-stripe** [comment]
> wondering if we should release it separately from your other PR in case this doesnt work for jfrog or dockers. but i could be just over-thinking

That's fair, I'll release the other one first.

**Review by etsai-stripe — APPROVED** [review]

**Pull request merged**